docs/user-guide.md

```diff
@@ -53,9 +53,9 @@ You will be prompted for a password to encrypt the database, keep the password s
 - You will see `→` in your message if it is not sent yet, `✓` if it was send, `✓✓` when the message was read, or `✖` if the message was not delivered to some of the recipients.
 - You can send emojis sending typing their string representation, for example: `:thumbs_up:`
 - You can have emoji, contact names and commands autocompletion typing `:`, `@` or `/` and pressing <kbd>Tab</kbd>
-- You can tweak the app colors editing `~/.curseddelta/theme.json`
-- You can tweak the app keymap editing `~/.curseddelta/keymap.json`
-- You can tweak the app settings editing `~/.curseddelta/curseddelta.conf`
+- You can tweak the app colors editing `$XDG_CONFIG_HOME/curseddelta/theme.json`
```

Comment:
this variable is not even defined in my Xubuntu, I don't think using it here is good, maybe `$HOME` but I think `~/` is clear enough, maybe add a note "if $XDG_CONFIG_HOME is defined it will be used instead of `~/` / `$HOME`

Reply:
good idea! tho it's strange that on xubuntu it's not set